torch/nn/functional.py

```diff
@@ -1904,13 +1904,15 @@ def embedding(
     else:
         padding_idx = -1
     if max_norm is not None:
+        # Note [embedding_renorm contiguous]
```

Comment:
I can't comment on this line because it's not in the review, but for the `padding_idx` docs above (line 1852), I think the wording is strange.

"initialized to zeros" doesn't really make sense in the functional context where `weight` is passed in. Imo the wording should be changed to indicate that the output is padded with an all-zero vector wherever padding_idx is encountered.